services/file_update.proto

```diff
@@ -31,37 +38,74 @@ import "timestamp.proto";
 import "google/protobuf/wrappers.proto";
 
 /**
- * Modify the metadata and/or contents of a file. If a field is not set in the transaction body, the
- * corresponding file attribute will be unchanged. This transaction must be signed by all the keys
- * in the top level of a key list (M-of-M) of the file being updated. If the keys themselves are
- * being updated, then the transaction must also be signed by all the new keys. If the keys contain
- * additional KeyList or ThresholdKey then M-of-M secondary KeyList or ThresholdKey signing
- * requirements must be meet If the update transaction sets the <tt>auto_renew_account_id</tt> field 
- * to anything other than the sentinel <tt>0.0.0</tt>, the key of the referenced account must sign.
+ * Update the metadata, and/or replace the content, of a file in the
+ * Hedera File Service (HFS).
+ *
+ * Any field, other than `fileID` which is not set (i.e. is null)
+ * in this message SHALL be ignored.<br/>
+ * If the `keys` list for the identified file is an empty `KeyList`, then
+ * this message MUST NOT set any field except `expirationTime`.
```

Comment:
```suggestion0
 * If the `keys` list for the identified file is an empty `KeyList` (because this file was
 * previously created or updated to have an empty `KeyList`), then the file is considered
 * immutable and this message MUST NOT set any field except `expirationTime`.
```

Reply:
Added.